services-toolkit/about.hbs.md

```diff
@@ -1,13 +1,40 @@
+The main capabilities on offer in TAP through services toolkit are:
+
+1. The classes and claims abstraction - providing a simple, but powerful, user experience to apps teams, while serving to promote a strong separation of concerns between apps teams and ops teams alike
+1. Dynamic provisioning of service instances - providing an ability for apps teams to create service instances that adhere to company policy on-demand and as needed, without incurring any untoward delay
+1. Seamless integration of almost any service, cloud-based or on-prem, into TAP with minimal configuration overhead - providing a near-limitless range of services to help boost developer productivity
+
+## Where to start
+
+If this is your first time working with services on TAP, then a great place to start is the [Consume services on Tanzu Application Platform](../getting-started/consume-services.hbs.md) walkthrough. This walkthrough runs through the basics and should leave you ready and wanting more! After which you can return here to the services toolkit component documentation to continue your services journey on TAP.
+
+## How this documentation is organized
+
+The services toolkit component documentation is loosely based around the framework laid out by https://documentation.divio.com/. As such, it primarily consists of [tutorials](./tutorials.hbs.md), [how-to guides](./how-to-guides.hbs.md), [explanations](./explanation.hbs.md) and [reference material](./reference.hbs.md). Which section to choose from will depend on what you are hoping to achieve. 
+
+* I'm hoping to learn - refer to the [tutorials](./tutorials.hbs.md) section
+* I'm hoping to gain a deeper understanding of services toolkit - refer to the [explanations](./explanation.hbs.md) section
+* I'm hoping to find a set of steps to solve a specific problem - refer to the [how-to guides](./how-to-guides.hbs.md) section
+* I'm hoping to find specific information about something - refer to the [reference material](./reference.hbs.md) section
+
+Per the documentation framework, tutorials and explanations are of most relevance when studying, while how-to guides and reference material are of most use while actively working. If you're still not sure, why not choose from this hand-picked selection of the most useful docs on offer.
+
+**For apps teams**
+
+* [Tutorial - Working with Bitnami Services](./tutorials/working-with-the-bitnami-services.hbs.md)
+
+**For ops teams**
+
+* [Tutorial - Setup Dynamic Provisioning of Service Instances](./tutorials/setup-dynamic-provisioning.hbs.md)
+* [Tutorial - Integrating Cloud Services (AWS, Azure, GCP, etc.) into Tanzu Application Platform](./tutorials/integrate-cloud-services-aws-azure-gcp-into-tap.hbs.md)
+* [How-to guide - Configure Dynamic Provisioning of AWS RDS Service Instances](./how-to-guides/configure-dynamic-provisioning-rds.hbs.md)
+
+**For everyone**
```

Comment:
Suggestion: The "For Everyone" bit might make more sense at the top so it isn't missed by the specific persona's.